docarray/typing/url/url_3d/point_cloud_url.py

```diff
@@ -60,4 +70,38 @@ class MyDoc(BaseDocument):
             mesh = self._load_trimesh_instance(force='mesh')
             point_cloud = np.array(mesh.sample(samples))
 
-        return parse_obj_as(NdArray, point_cloud)
+        points = parse_obj_as(NdArray, point_cloud)
+        return PointsAndColors(points=points, colors=None)
+
+    def display(self, samples: int = 10000) -> None:
+        """
+        Plot point cloud from url.
+        :param samples: number of points to sample from the mesh.
+        """
+        tensors = self.load(samples=samples)
+        _display_point_cloud(points=tensors.points)
+
+
+def _display_point_cloud(
```

Comment:
lets define `_display_point_cloud` in the PointsAndColor document as a classmethod then you cast tensors.points into a PointsAndColor document and call dispaly